client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,19 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
```

Comment:
Hmm, maybe leave it for now. We're already renaming `LayoutLoggedOut`~~`Design`~~, so that maybe enough for the scope of this one.